src/scss/styles.scss

```diff
@@ -100,58 +90,61 @@ canvas {
 }
 
 .data-heading {
-  @include margin(0px, 0px, 15px, 0px);
-  @include font-size(1.5em);
-  @include font-weight(200);
+  @include margin(5px, 0px, 15px, 10px);
+  @include font-look(6em, 200);
   color: $header-font-color;
   letter-spacing: 1px;
 }
 
 .data-article {
-  @include font-weight(300);
-  // @include font-weight(.8em);
   @include margin(40px, 0px, 40px, 0px);
   text-align: center;
 }
 
 .data-text {
   @include margin(15px, 15px, 15px, 15px);
+  @include font-look(3.5em, 300);
 }
 
 .subtext {
-  @include font-size(10px);
-  @include font-weight(700);
-  @include margin(5px, 5px, 5px, 5px);
+  @include font-look(3em, 700);
+  @include margin(25px, 5px, 25px, 5px);
   color: $header-font-color;
 }
 
-.water-consumed {
-  @include margin(5px, 5px, 5px, 5px);
-}
-
 .number-of-steps-day {
-  @include margin(20px, 20px, 20px, 20px);
-  width: 100px;
-  height: 100px;
-  position: relative;
-  text-align: center;
+  @include margin(20px, auto, 50px, auto);
+  width: 280px;
+  @include font-look(1.2em, 200);
 }
 
 .number-of-steps-goal {
   @include margin(10px, 10px, 10px, 10px);
```

Comment:
These margin and padding mixins are probably not great use-cases for a mixin. You're actually writing MORE code by breaking these into mixins rather than less. e.g. you can do the same logic with a single line of CSS shorthand:

`margin: 10px 10px 10px 10px`

And this will set the top/right/bottom/left margins of the element with a single line of code. Mixins are better used when there are many properties you want to specify about that content, rather than just one.